GameManipulator.js

```diff
@@ -228,6 +228,19 @@ GameManipulator.startNewGame = function (next) {
 
 }
 
+// reload the page
+GameManipulator.reloadPage = function ()
+{
+  // retrieves platform
+  var platform = process.platform;
+
+  if(/^win/.test(process.platform)) {
```

Comment:
If you make this an `else`, it will work in Linux. 

FYI OS X is the only OS (that I know of) with strange keyboard shortcuts. If it works in Windows (outside of windows-specific activities obviously), then it works on Linux.